frontend/src/components/Admin/AddUser/AddUser.tsx

```diff
@@ -125,6 +146,46 @@ export default function AddUser(): JSX.Element {
     });
   }
 
+  async function uploadUsersCsv(csvFile: File | undefined): Promise<void> {
+    if (csvFile) {
+      const csv = await csvFile.text();
+      const json = await csvtojson().fromString(csv);
+      const allNetIds = allMembers.map((m) => m.netid);
+      const members: IdolMember[] = [];
+      if (json[0].email && json[0].role) {
+        json.forEach((m) => {
+          const netId = getNetIDFromEmail(m.email);
+          const currMember = allMembers.find((mem) => mem.netid === netId);
+          if (currMember) {
+            const member = {};
+            Object.keys(EMPTY_MEMBER).forEach((field) => {
+              member[field] = m[field] ?? currMember[field];
+            });
+            members.push(member);
+          } else {
+            const member = { ...EMPTY_MEMBER };
+            Object.keys(EMPTY_MEMBER).forEach((field) => {
+              if (m[field]) member[field] = m[field];
+            });
+            members.push(member);
```

Comment:
I think these member objects are missing the `netid` and `roleDescription` fields.

Reply:
Yep, those two fields are missing.

<img width="965" alt="Screenshot 2023-09-19 at 12 33 00 PM" src="https://github.com/cornell-dti/idol/assets/59291082/fc3ffab0-8ba5-4aed-a08d-3dfef2de19f5">

Reply:
You can insert these fields right before you push the `member` to `members` array (or if you make the revision I said above, just add the two fields right before you call `MembersAPI`.